### PR TITLE
project-library Viktoria O

### PR DESCRIPTION
**Pull request by viktoria-olo**
## Netlify link
https://main--sparkling-cajeta-657abb.netlify.app/

## Collaborators
No collaborators

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,406 +1,395 @@
...
     rating: 4.12,
     description:
-      'A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.',
-    image: './books-images/unknown.jpg'
+      "A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.",
+    image: "./books-images/unknown.jpg",
+  },
+];
+
+///////////////////////////// DOM selectors /////////////////////////////
```
Comment (johannacatalinismith) [inline script.js:184]: Really like how you're doing clear comments about what goes where.

File: script.js [hunk 2]
```diff
@@ -1,406 +1,395 @@
...
+  const filteredBooks = filterByGenre(books, genre);
+  displayBooks(filteredBooks);
+  nowDispBooks = filteredBooks;
+};
+
+// Function to get a random number
+const getRandomNum = (min, max) => {
+  // prettier-ignore
+  let getRandom = Math.floor((Math.random() * (max - min + 1) + min));
+  return getRandom;
```
Comment (johannacatalinismith) [inline script.js:270]: This variable name is a verb, it could maybe be mistaken for being a function. Maybe instead name it randomNumber, for example.

**Review by johannacatalinismith — COMMENTED** [review]
On a overall i'm very impressed! Good job!

**Review by NellaMori — APPROVED** [review]
Hi Viktoria, I only have one word for you:✨  bellissimo ✨ You worked really really well, followed the instructions, guidelines to write good code, made the app responsive and used filters properly combining them to the genre selection. Good job with the app UI and CSS. Hat off 🎩  looking forward to seeing you continuing like this 💪

**Pull request closed without merging**